site/pages/review-disruption/[disruptionId].page.tsx

```diff
@@ -413,6 +427,24 @@ const ReviewDisruption = ({ disruption, previousSocialMediaPosts, csrfToken }: R
                         <button className="govuk-button mt-8" data-module="govuk-button">
                             Publish disruption
                         </button>
+                        <button
+                            className="govuk-button govuk-button--warning ml-5 mt-8"
+                            data-module="govuk-button"
+                            onClick={(e) => {
+                                e.preventDefault();
+                                deleteActionHandler(disruption.disruptionId, disruption.summary);
+                            }}
+                        >
+                            Delete disruption
+                        </button>
+                        {popUpState && csrfToken ? (
```

Comment:
I'd maybe move this outside the CsrfForm element since we should avoid nesting forms where possible

Reply:
done